src/Symfony/Component/Lock/Store/DoctrineDbalPostgreSqlStore.php

```diff
@@ -278,7 +278,7 @@ private function filterDsn(string $dsn): string
         }
 
         [$scheme, $rest] = explode(':', $dsn, 2);
-        $driver = strtok($scheme, '+');
+        $driver = substr($scheme, 0, strpos($scheme, '+') ?: null);
```

Comment:
passing null as 3rd arg works only since php 8 :'(
https://php.net/substr

Reply:
Oh, I missed that subtlety... Thank you 👍